axelrod/tests/unit/test_worse_and_worse.py

```diff
@@ -0,0 +1,97 @@
+"""Test for the Worse and Worse strategies."""
+
+import axelrod
+
+from .test_player import TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+class TestWorseAndWorse(TestPlayer):
+
+    name = "Worse and worse"
+    player = axelrod.WorseAndWorse
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        """
+        Test that the strategy gives expected behaviour
+        """
+
+        axelrod.seed(1)
+        opponent = axelrod.Cooperator()
+        player = axelrod.WorseAndWorse()
+        match = axelrod.Match((opponent, player), turns=5)
+        self.assertEqual(match.play(), [('C', 'C'),
```

Comment:
Once you've made the expected change, this might change but either way it would be good if you picked a seed that didn't just have `WorseAndWorse` cooperate (ie choose one that gives a mix of C and D). Change the strategy first though.